src/test/java/com/rtb/projectmanagementtool/user/UserControllerTest.java

```diff
@@ -0,0 +1,58 @@
+package com.rtb.projectmanagementtool.user;
+
+import static com.google.appengine.api.datastore.FetchOptions.Builder.withLimit;
+
+import com.google.appengine.api.datastore.*;
+import com.google.appengine.api.datastore.Query.*;
+import com.google.appengine.api.datastore.dev.LocalDatastoreService;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import java.util.Arrays;
+import java.util.HashSet;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class UserControllerTest {
+  private HashSet<UserData> testUsers;
```

Comment:
The way that this is setup, it looks like it'll be a list of users that's accessible throughout the whole test class.  For building test cases, it would be best to avoid shared resources between the different test methods if possible.  This means if you have test `testAddUsers` that adds items into `testUsers`, then later on you have some other test (let's say 'loadUsersTest') that looks at `testUsers`, you'd end up affecting your `loadUsersTest` test with results from your `testAddUsers` test.